spring-integration-core/src/main/java/org/springframework/integration/leader/event/LeaderEventPublisher.java

```diff
@@ -45,4 +46,14 @@
 	 */
 	void publishOnRevoked(Object source, Context context, String role);
 
+	/**
+	 * Publish a failure to acquire lock event.
+	 *
+	 * @param source the component generated this event
+	 * @param context the context associated with event
+	 * @param role the role of the leader
+	 */
```

Comment:
Need `@since 5.0`
And no nay blank lines in the method JavaDocs.
All the affected classes should be updated for the current year in their Copyright.

(I can fix all on merge 😄 )

Reply:
:thumbsup: